lighthouse-core/lib/traces/tracing-processor.js

```diff
@@ -237,6 +237,20 @@ class TraceProcessor {
       }
     }
 
+    // Support the case where everything else fails.
```

Comment:
can you add more detail about the event we're looking for (e.g. are the large number of conditions in `events.find()` due to them being optional or because of tsc or is it that there are a bunch of `navigationStart` events and the one we're interested in has all these defined?)

Reply:
SG 👍